### DISCUSSION TITLE
작업기간은 어느 정도로 생각하고 계신가요?

### DISCUSSION THREAD

**Discussion by kycasdzxc** [josa-moa/josa-moa#23] [Q&A] [answered]
작업기간은 어느 정도로 생각하고 계신가요?

**Comment by im-dullin** [comment, marked as answer]
> 작업기간은 어느 정도로 생각하고 계신가요?

현재 계획중인 작업기간으로는 1-2 개월이며, 추후 더해질 기능을 계획 후 지속적으로 보완할 예정입니다.
